gpu/gpu.go

```diff
@@ -242,6 +242,15 @@ func getCPUMem() (memInfo, error) {
 }
 
 func CheckVRAM() (int64, error) {
+	userLimit := os.Getenv("OLLAMA_MAX_VRAM")
```

Comment:
do we want to have this temp/debug env var for darwin as well?

Reply:
if helpful, here's the only place where it's called and used https://github.com/ollama/ollama/blob/0ded7fdc4b33801fd0115656927b6097f800c544/llm/llm.go#L47

Reply:
Good point - let me add that to the darwin file.